notebooks/tutorial_ivf_pq.ipynb

```diff
@@ -0,0 +1,1226 @@
+{
```

Comment:
<p>Again, these charts look pretty mangled with overlapping labels. Since the image is often encoded as base64, I'm assuming it'll look the same for the user as it does on ReviewNB. These should be styled a little bit or made more clear. Right now it just looks like a bunch of unreadable overlapping text. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/raft/pull/1544/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>